tracardi/process_engine/action/v1/flow/start/start_action.py

```diff
@@ -73,6 +73,8 @@ def register() -> Plugin:
             keywords=['start node'],
             type="startNode",
             icon='start',
+            width=200,
```

Comment:
why the start node was changed with this PR?

Reply:
This is due to the difference in shape on front-end between Start node and other nodes - had to be included.